core/server/models/index.js

```diff
@@ -1,61 +1,71 @@
-    // module can safely access models without fear of introducing circular
-    // dependency issues.
-    // @returns {Promise}
-    init: function init() {
-        var self = this;
-
-        // One off inclusion of Base file.
-        self.Base = require('./base');
-
-        // Require all files in this directory
-        return requireTree.readAll(__dirname, {followSymlinks: false}).then(function then(modelFiles) {
-            // For each found file, excluding those we don't want,
-            // we will require it and cache it here.
-            _.each(modelFiles, function each(path, fileName) {
-                // Return early if this fileName is one of the ones we want
-                // to exclude.
-                if (_.contains(self.excludeFiles, fileName)) {
-                    return;
-                }
-
-                // Require the file.
-                var file = require(path);
-
-                // Cache its `export` object onto this object.
-                _.extend(self, file);
-            });
-
-            return;
-        });
-    },
-    // ### deleteAllContent
-    // Delete all content from the database (posts, tags, tags_posts)
-    deleteAllContent: function deleteAllContent() {
-        var self = this;
-
-        return self.Post.findPage({limit: 'all'}).then(function then(data) {
-            return Promise.all(_.map(data.posts, function mapper(post) {
-                return self.Post.destroy({id: post.id});
+/**
+ * Expose all models
+ */
+
+exports = module.exports;
+
+models = [
+    'accesstoken',
+    'app-field',
+    'app-setting',
+    'app',
+    'client-trusted-domain',
+    'client',
+    'permission',
+    'post',
+    'refreshtoken',
+    'role',
+    'settings',
+    'tag',
+    'user'
+];
+
+function init() {
+    exports.Base = require('./base');
+
+    models.forEach(function (name) {
+        _.extend(exports, require('./' + name));
+    });
+
+    return Promise.resolve();
+}
+
+/**
+ * TODO: move to some other place
+ */
+
+// ### deleteAllContent
+// Delete all content from the database (posts, tags, tags_posts)
+exports.deleteAllContent = function deleteAllContent() {
+    var self = exports;
+
+    return self.Post.findAll().then(function then(posts) {
+        return Promise.all(_.map(posts.toJSON(), function mapper(post) {
+            return self.Post.destroy({id: post.id});
+        }));
+    }).then(function () {
+        return self.Tag.findAll().then(function then(tags) {
+            return Promise.all(_.map(tags.toJSON(), function mapper(tag) {
+                return self.Tag.destroy({id: tag.id});
             }));
-        }).then(function () {
-            return self.Tag.findPage({limit: 'all'}).then(function then(data) {
-                return Promise.all(_.map(data.tags, function mapper(tag) {
-                    return self.Tag.destroy({id: tag.id});
-                }));
-            });
         });
```

Comment:
This part looks like the rebase might have gone wrong - `.findAll` has been removed :wink: